source/cpptooling/data/representation.d

```diff
@@ -1381,14 +1382,6 @@ const:
     }
 }
 
-/// Dictates how the namespaces are merged.
-enum MergeMode {
-    /// Merge everything except nested namespaces.
-    shallow,
-    /// Merge everything.
-    full
-}
-
 @safe struct CppNamespace {
```

Comment:
What are the tests that you ran that started to fail after the merge?
I'm thinking of looking into the problem but need to be able to reproduce it.